Automattic/wp-calypso
# composite-checkout: Remove renderDisplayValueMarkdown

#### Changes proposed in this Pull Request

The `renderDisplayValueMarkdown()` helper function in the `@automattic/composite-checkout` package was intended as a way to more easily display crossed-out prices for discounts, but in practice that data is being rendered live by a custom line item display. There's also some evidence that there may be a bug in that function which could sometimes cause a fatal React error.

This PR removes that function from the package and all uses in calypso.

#### Testing instructions

- Visit composite checkout with a product that has a discounted price (eg: a domain with a plan or a plan with a coupon) and verify that the original price shows as crossed-out and the actual price displays as expected.
